public/js/passwordindicator.js

```diff
@@ -0,0 +1,59 @@
+const indicator = document.querySelector(".indicator");
```

Comment:
Use a better name for classes so that it doesn't conflict with some other variables in future.

like instead of `indicator` use `pass-security-indicator`